catalog/core/catalog-core-subscriptionstore/subscriptionstore-impl/pom.xml

```diff
@@ -37,6 +37,31 @@
             <artifactId>subscriptionstore-internal-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>javax.cache</groupId>
+            <artifactId>cache-api</artifactId>
+            <version>1.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.cache2k</groupId>
+            <artifactId>cache2k-all</artifactId>
```

Comment:
Could prevent typos, also not sure you would ever want different versions of these dependencies.